ui/src/components/AppDetailedToolbar.js

```diff
@@ -24,7 +24,7 @@ export class AppDetailedToolbar extends React.Component {
         </ToolbarGroup>
         <ToolbarGroup className='toolbar-buttons'>
           <Button className='toolbar-button' onClick={onDisableApp} variant="primary">Disable App</Button>
-          <Button className='toolbar-button' onClick={onSaveApp} variant="primary">Save</Button>
+          <Button className='toolbar-button' onClick={() => onSaveApp(true)} variant="primary">Save</Button>
```

Comment:
Should we move the listener implementation out into it's own method? 

Implication of defining it here is that it will be defined every time the component re-renders. Placing it in it's own method in the component means the handler will be re-used instead of being recreated each time.

Reply:
This is interesting. Do you have any reading material so I can find out more of the pros/cons of what you are suggesting?

Reply:
Actually I was slightly incorrect above - what happens is a new callback is created each time, which if you're passing that callback as a prop to children components can cause them to re-render. That's probably not the case here, as we're just passing it to a PF4 Button component (which probably isn't composed of other children components). 

Snippet from [React docs](https://reactjs.org/docs/handling-events.html):

```
The problem with this syntax is that a different callback is created each time the LoggingButton renders. In most cases, this is fine. However, if this callback is passed as a prop to lower components, those components might do an extra re-rendering. We generally recommend binding in the constructor or using the class fields syntax, to avoid this sort of performance problem.
```

Reply:
Fantastic, thank you @damienomurchu . This is good to know. I will be conscious of it during the refactoring to ensure it is not passed as a prop to child components.

Reply:
Yeah, its one of those small potential gotcha's that probably justifies not using inline listeners if only so that you do not need to monitor for edge cases

Reply:
perfect, I will modify this on the refactor. are you happy to approve the PR?

Reply:
Sure, let me just quickly verify the functionality & will approve then :+1: